telemetry/newrelic/src/main/scala/io/buoyant/telemetry/newrelic/NewRelicTelemeter.scala

```diff
@@ -0,0 +1,107 @@
+package io.buoyant.telemetry.newrelic
+
+import com.fasterxml.jackson.annotation.{JsonFormat, JsonValue}
+import com.twitter.conversions.time._
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{MediaType, Method, Request, Response}
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.NullTracer
+import com.twitter.util._
+import io.buoyant.config.Parser
+import io.buoyant.telemetry.Metric.{Counter, Gauge, Stat, None => MetricNone}
+import io.buoyant.telemetry.{MetricsTree, Telemeter}
+import java.util.concurrent.atomic.AtomicBoolean
+import com.twitter.logging.Logger
+
+class NewRelicTelemeter(
```

Comment:
How are we testing this class?

Reply:
Added tests for initializers in 7fdfa1d, will add tests for telemeter reporting as well